ui/app/templates/rules.html

```diff
@@ -287,6 +287,7 @@ <h5 title="Fallback Mapping" ng-show="rule.fallbackMapping || (show_sc && rule.s
             <strong>Note!</strong> Fallback Mapping is unused when Background Rate is 100
           </span>
         </span>
+        <rulefield showdiff="!rule_id" showsc="show_sc" rule="rule" fieldname="has_wnp" fieldtitle="WNP?"></rulefield>
```

Comment:
We need to do this for both `mapping` and `fallbackMapping`. How about some sort of badge that appears next to them? Something like: https://screenshotscdn.firefoxusercontent.com/images/8c7569c2-1e9d-48d0-9d9d-86981d9c13a7.png -- except with a nicer background colour.

Reply:
Oops! hadn't noticed that both `mapping` and `fallbackMapping` are shown to the user in cases where also `mapping` exists. Going to fix this. Thanks 👍

Reply:
Using badge is also a good idea. What colors do you have in mind? What do you think about blue? I'm not so good with colors 😄

Reply:
Blue seems fine :). We can always change it later if we don't like it!